### ISSUE TITLE
uninitialized constant Gym::CodeSigningMapping

### ISSUE THREAD

**Issue by robwalkerco** [fastlane/fastlane#10098]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description
When running `gym` as part of a lane I consistently get an `uninitialized constant Gym::CodeSigningMapping` error, but havent been able to find any cause.

I have also been having another issue when using fastlane 2.51.0 where gym is not creating an ipa file for my react native (9.42.2) project, but not sure if that related or not!

##### Complete output when running fastlane, including the stack trace and command used
> You can use: `--capture_output` as the last commandline argument to get that collected for you

```
Roberts-MacBook-Pro:papertrailApp robertwalker$ fastlane ios beta
[22:57:20]: -------------------------------------------------
[22:57:20]: --- Step: Verifying required fastlane version ---
[22:57:20]: -------------------------------------------------
[22:57:20]: Your fastlane version 2.53.1 matches the minimum requirement of 2.51.0  ✅
[22:57:20]: ------------------------------
[22:57:20]: --- Step: default_platform ---
[22:57:20]: ------------------------------
[22:57:20]: Driving the lane 'ios beta' 🚀
[22:57:20]: -----------------------
[22:57:20]: --- Step: cocoapods ---
[22:57:20]: -----------------------
[22:57:20]: $ cd './ios' && bundle exec pod install
[22:57:22]: ▸ Analyzing dependencies
[22:57:22]: ▸ Downloading dependencies
[22:57:22]: ▸ Using Crashlytics (3.8.4)
[22:57:22]: ▸ Using Fabric (1.6.11)
[22:57:22]: ▸ Using Intercom (3.2.5)
[22:57:22]: ▸ Generating Pods project
[22:57:22]: ▸ Integrating client project
[22:57:23]: ▸ Sending stats
[22:57:23]: ▸ Pod installation complete! There are 3 dependencies from the Podfile and 3 total pods installed.
[22:57:23]: --------------------
[22:57:23]: --- Step: prompt ---
[22:57:23]: --------------------
[22:57:23]: Changelog:
[22:57:23]: Submit inputs using "END"
END
[22:57:29]: --------------------
[22:57:29]: --- Step: prompt ---
[22:57:29]: --------------------
[22:57:29]: Should the build be distributed to external testers? (y/n)


Please enter "yes" or "no".
[22:57:29]: Should the build be distributed to external testers? (y/n)
n
[22:57:30]: -------------------
[22:57:30]: --- Step: match ---
[22:57:30]: -------------------
[22:57:30]: Successfully loaded '/Users/robertwalker/code/papertrailApp/fastlane/Matchfile' 📄

+----------------+----------------------------------------------+
|          Detected Values from './fastlane/Matchfile'          |
+----------------+----------------------------------------------+
| git_url        | git+ssh://git@github.com/papertrailio/ios-a  |
|                | pp-certs.git                                 |
| shallow_clone  | false                                        |
| platform       | ios                                          |
| app_identifier | com.papertrail.app                           |
| username       | ios@papertrail.io                            |
+----------------+----------------------------------------------+


+-----------------------+----------------------------------------------+
|                       Summary for match 2.53.1                       |
+-----------------------+----------------------------------------------+
| type                  | appstore                                     |
| git_url               | git+ssh://git@github.com/papertrailio/ios-a  |
|                       | pp-certs.git                                 |
| shallow_clone         | false                                        |
| platform              | ios                                          |
| app_identifier        | com.papertrail.app                           |
| username              | ios@papertrail.io                            |
| git_branch            | master                                       |
| keychain_name         | login.keychain                               |
| readonly              | false                                        |
| team_id               | Z6Y4X6J5N8                                   |
| verbose               | false                                        |
| force                 | false                                        |
| skip_confirmation     | false                                        |
| clone_branch_directly | false                                        |
| force_for_new_devices | false                                        |
| skip_docs             | false                                        |
+-----------------------+----------------------------------------------+

[22:57:30]: Cloning remote git repo...
[22:57:30]: If cloning the repo takes too long, you can use the `clone_branch_directly` option in match.
[22:57:32]: 🔓  Successfully decrypted certificates repo
[22:57:32]: Verifying that the certificate and profile are still valid on the Dev Portal...
[22:57:35]: Installing certificate...

+-------------------+-----------------------------------------------+
|                       Installed Certificate                       |
+-------------------+-----------------------------------------------+
| User ID           | Z6Y4X6J5N8                                    |
| Common Name       | iPhone Distribution: Kit Log Ltd (Z6Y4X6J5N8) |
| Organisation Unit | Z6Y4X6J5N8                                    |
| Organisation      | Kit Log Ltd                                   |
| Country           | US                                            |
| Start Datetime    | May 11 21:21:22 2017 GMT                      |
| End Datetime      | May 11 21:21:22 2018 GMT                      |
+-------------------+-----------------------------------------------+

[22:57:36]: Installing provisioning profile...

+---------------------+------------------------------+------------------------------+
|                          Installed Provisioning Profile                           |
+---------------------+------------------------------+------------------------------+
| Parameter           | Environment Variable         | Value                        |
+---------------------+------------------------------+------------------------------+
| App Identifier      |                              | com.papertrail.app           |
| Type                |                              | appstore                     |
| Platform            |                              | ios                          |
| Profile UUID        | sigh_com.papertrail.app_app  | d49e2942-7d6e-468a-b996-f3c  |
|                     | store                        | 27886bb53                    |
| Profile Name        | sigh_com.papertrail.app_app  | match AppStore               |
|                     | store_profile-name           | com.papertrail.app           |
| Profile Path        | sigh_com.papertrail.app_app  | /Users/robertwalker/Library  |
|                     | store_profile-path           | /MobileDevice/Provisioning   |
|                     |                              | Profiles/d49e2942-7d6e-468a  |
|                     |                              | -b996-f3c27886bb53.mobilepr  |
|                     |                              | ovision                      |
| Development Team ID | sigh_com.papertrail.app_app  | Z6Y4X6J5N8                   |
|                     | store_team-id                |                              |
+---------------------+------------------------------+------------------------------+

[22:57:37]: All required keys, certificates and provisioning profiles are installed 🙌
[22:57:37]: Setting Provisioning Profile type to 'app-store'
[22:57:37]: ------------------------------------
[22:57:37]: --- Step: increment_build_number ---
[22:57:37]: ------------------------------------
Current version of project papertrailApp is:
    135

/Users/robertwalker/code/papertrailApp
[22:57:37]: $ cd /Users/robertwalker/code/papertrailApp/ios && agvtool next-version -all && cd -
[22:57:38]: ▸ Setting version of project papertrailApp to:
[22:57:38]: ▸ 136.
[22:57:38]: ▸ Also setting CFBundleVersion key (assuming it exists)
[22:57:38]: ▸ Updating CFBundleVersion in Info.plist(s)...
[22:57:38]: ▸ Updated CFBundleVersion in "papertrailApp.xcodeproj/../papertrailApp-tvOS/Info.plist" to 136
[22:57:38]: ▸ Updated CFBundleVersion in "papertrailApp.xcodeproj/../papertrailApp-tvOSTests/Info.plist" to 136
[22:57:38]: ▸ Updated CFBundleVersion in "papertrailApp.xcodeproj/../papertrailApp/Info.plist" to 136
[22:57:38]: ▸ Updated CFBundleVersion in "papertrailApp.xcodeproj/../papertrailAppTests/Info.plist" to 136
[22:57:38]: ▸ Updated CFBundleVersion in "papertrailApp.xcodeproj/../papertrailAppUITests/Info.plist" to 136
[22:57:38]: ▸ /Users/robertwalker/code/papertrailApp
[22:57:39]: --------------------------------
[22:57:39]: --- Step: get_version_number ---
[22:57:39]: --------------------------------
[22:57:39]: $ cd /Users/robertwalker/code/papertrailApp/ios && agvtool what-marketing-version -terse
[22:57:40]: ▸ "papertrailApp.xcodeproj/../papertrailApp-tvOS/Info.plist"=2.3.1
[22:57:40]: ▸ "papertrailApp.xcodeproj/../papertrailApp-tvOSTests/Info.plist"=2.3.1
[22:57:40]: ▸ "papertrailApp.xcodeproj/../papertrailApp/Info.plist"=2.3.1
[22:57:40]: ▸ "papertrailApp.xcodeproj/../papertrailAppTests/Info.plist"=2.3.1
[22:57:40]: ▸ "papertrailApp.xcodeproj/../papertrailAppUITests/Info.plist"=2.3.1
[22:57:40]: -----------------
[22:57:40]: --- Step: gym ---
[22:57:40]: -----------------
+------------------------------------+-----------------------------------------+
|                                 Lane Context                                 |
+------------------------------------+-----------------------------------------+
| DEFAULT_PLATFORM                   | ios                                     |
| PLATFORM_NAME                      | ios                                     |
| LANE_NAME                          | ios beta                                |
| SIGH_PROFILE_TYPE                  | app-store                               |
| MATCH_PROVISIONING_PROFILE_MAPPING | {"com.papertrail.app"=>"match AppStore  |
|                                    | com.papertrail.app"}                    |
| BUILD_NUMBER                       | 136                                     |
| VERSION_NUMBER                     | 2.3.1                                   |
+------------------------------------+-----------------------------------------+
[22:57:40]: uninitialized constant Gym::CodeSigningMapping

+------+------------------------------+-------------+
|                 fastlane summary                  |
+------+------------------------------+-------------+
| Step | Action                       | Time (in s) |
+------+------------------------------+-------------+
| 1    | Verifying required fastlane  | 0           |
|      | version                      |             |
| 2    | default_platform             | 0           |
| 3    | cocoapods                    | 2           |
| 4    | prompt                       | 5           |
| 5    | prompt                       | 1           |
| 6    | match                        | 6           |
| 7    | increment_build_number       | 2           |
| 8    | get_version_number           | 1           |
| 💥   | gym                          | 0           |
+------+------------------------------+-------------+

[22:57:40]: fastlane finished with errors

Looking for related GitHub issues on fastlane/fastlane...

Found no similar issues. To create a new issue, please visit:
https://github.com/fastlane/fastlane/issues/new
Run `fastlane env` to append the fastlane environment to your issue
/Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/fastlane-2.53.1/fastlane/lib/fastlane/actions/gym.rb:29:in `run': [!] uninitialized constant Gym::CodeSigningMapping (NameError)
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/fastlane-2.53.1/fastlane/lib/fastlane/runner.rb:252:in `block (2 levels) in execute_action'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/fastlane-2.53.1/fastlane/lib/fastlane/actions/actions_helper.rb:50:in `execute_action'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/fastlane-2.53.1/fastlane/lib/fastlane/runner.rb:230:in `block in execute_action'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/fastlane-2.53.1/fastlane/lib/fastlane/runner.rb:226:in `chdir'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/fastlane-2.53.1/fastlane/lib/fastlane/runner.rb:226:in `execute_action'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/fastlane-2.53.1/fastlane/lib/fastlane/runner.rb:148:in `trigger_action_by_name'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/fastlane-2.53.1/fastlane/lib/fastlane/fast_file.rb:146:in `method_missing'
	from Fastfile:116:in `block (2 levels) in parsing_binding'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/fastlane-2.53.1/fastlane/lib/fastlane/lane.rb:33:in `call'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/fastlane-2.53.1/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/fastlane-2.53.1/fastlane/lib/fastlane/runner.rb:45:in `chdir'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/fastlane-2.53.1/fastlane/lib/fastlane/runner.rb:45:in `execute'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/fastlane-2.53.1/fastlane/lib/fastlane/lane_manager.rb:52:in `cruise_lane'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/fastlane-2.53.1/fastlane/lib/fastlane/command_line_handler.rb:30:in `handle'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/fastlane-2.53.1/fastlane/lib/fastlane/commands_generator.rb:104:in `block (2 levels) in run'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/commander-fastlane-4.4.5/lib/commander/command.rb:178:in `call'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/commander-fastlane-4.4.5/lib/commander/command.rb:153:in `run'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/commander-fastlane-4.4.5/lib/commander/runner.rb:476:in `run_active_command'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/fastlane-2.53.1/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:64:in `run!'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/commander-fastlane-4.4.5/lib/commander/delegates.rb:15:in `run!'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/fastlane-2.53.1/fastlane/lib/fastlane/commands_generator.rb:303:in `run'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/fastlane-2.53.1/fastlane/lib/fastlane/commands_generator.rb:42:in `start'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/fastlane-2.53.1/fastlane/lib/fastlane/cli_tools_distributor.rb:66:in `take_off'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/gems/fastlane-2.53.1/bin/fastlane:20:in `<top (required)>'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/bin/fastlane:23:in `load'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/bin/fastlane:23:in `<main>'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/bin/ruby_executable_hooks:15:in `eval'
	from /Users/robertwalker/.rvm/gems/ruby-2.3.1/bin/ruby_executable_hooks:15:in `<main>'

### Environment

Please run `fastlane env` and copy the output below. This will help us help you :+1:
If you used `--capture_output` option please remove this block - as it is already included there.

<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.12.4                                     |
| Ruby                        | 2.3.1                                       |
| Bundler?                    | true                                        |
| Git                         | git version 2.6.2                           |
| Installation Source         | ~/.rvm/gems/ruby-2.3.1/bin/fastlane         |
| Host                        | Mac OS X 10.12.4 (16E195)                   |
| Ruby Lib Dir                | ~/.rvm/rubies/ruby-2.3.1/lib                |
| OpenSSL Version             | OpenSSL 1.0.2h  3 May 2016                  |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 8.3.3                                       |


### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_GB.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |


### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
# Customise this file, documentation can be found here:
# https://github.com/fastlane/fastlane/tree/master/fastlane/docs
# All available actions: https://docs.fastlane.tools/actions
# can also be listed using the `fastlane actions` command

# Change the syntax highlighting to Ruby
# All lines starting with a # are ignored when running `fastlane`

# If you want to automatically update fastlane if a new version is available:
# update_fastlane

# This is the minimum version number required.
# Update this, if you use features of a newer version
fastlane_version "2.51.0"

default_platform :ios

platform :ios do
  before_all do
    # ENV["SLACK_URL"] = "https://hooks.slack.com/services/..."
    cocoapods(
      podfile: './ios'
    )

  end

  desc "Increment the app version patch"
  lane :bumpPatch do
    version_number = increment_version_number(
      xcodeproj: './ios/papertrailApp.xcodeproj',
      bump_type: "patch"
    )

    commit_version_bump(
      xcodeproj: './ios/papertrailApp.xcodeproj',
      message: "build(ios): bump patch to #{version_number}",
      force: true
    )

    push_to_git_remote
  end

  desc "Increment the app version minor"
  lane :bumpMinor do
    version_number = increment_version_number(
      xcodeproj: './ios/papertrailApp.xcodeproj',
      bump_type: "minor"
    )

    commit_version_bump(
      xcodeproj: './ios/papertrailApp.xcodeproj',
      message: "build(ios): bump minor to #{version_number}",
      force: true
    )

    push_to_git_remote
  end

  desc "Increment the app version major"
  lane :bumpMajor do
    ensure_git_branch # defaults to `master` branch

    version_number = increment_version_number(
      xcodeproj: './ios/papertrailApp.xcodeproj',
      bump_type: "major"
    )

    commit_version_bump(
      xcodeproj: './ios/papertrailApp.xcodeproj',
      message: "build(ios): bump major to #{version_number}",
      force: true
    )

    push_to_git_remote
  end

  # desc "Submit a new Beta Build to Apple TestFlight"
  # desc "This will also make sure the profile is up to date"
  # lane :beta do
  #   # match(type: "appstore") # more information: https://codesigning.guide
  #   gym(scheme: "papertrailApp") # Build your app - more options available
  #   pilot
  #
  #   # sh "your_script.sh"
  #   # You can also use other beta testing services here (run `fastlane actions`)
  # end

  desc "Submit a new Beta Build to Apple TestFlight"
  desc "This action will also do a build version bump and push it to git."
  desc "This will also make sure the profile is up to date"
  desc "This will also send an email to all testers."
  lane :beta do

    changelog = prompt(
      text: "Changelog: ",
      multi_line_end_keyword: "END"
    )

    distribute_external = prompt(
      text: "Should the build be distributed to external testers?",
      boolean: true
    )

    match(type: "appstore")

    # sh %Q{cd ../ && rm -R node_modules && yarn}

    build_number = increment_build_number(
      xcodeproj: './ios/papertrailApp.xcodeproj',
    )

    version_number = get_version_number(
      xcodeproj: './ios/papertrailApp.xcodeproj',
    )

    gym(
      scheme: "papertrailAppRelease",
      workspace: './ios/papertrailApp.xcworkspace'
    )

    pilot(
      changelog: changelog,
      distribute_external: distribute_external
    )

    new_sentry_release(
      build_number: build_number,
      version_number: version_number
    )

    commit_version_bump(
      xcodeproj: './ios/papertrailApp.xcodeproj',
      message: "build(ios): push #{version_number}: #{build_number} to beta",
      force: true
    )

    add_git_tag

    push_to_git_remote

  end

  desc "Deploy a new version to the App Store"
  lane :release do
    # ensure_git_branch # defaults to `master` branch

    update_screenshots = prompt(
      text: "Update screenshots?",
      boolean: true
    )

    match(type: "appstore")

    if update_screenshots
      snapshot

      frameit(
        silver: true
      )
    end

    build_number = increment_build_number(
      xcodeproj: './ios/papertrailApp.xcodeproj',
    )

    version_number = get_version_number(
      xcodeproj: './ios/papertrailApp.xcodeproj',
    )

    gym(
      scheme: "papertrailAppRelease",
      workspace: './ios/papertrailApp.xcworkspace'
    )

    ENV["DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS"] = "-t DAV"
    deliver(
      force: true,
      automatic_release: true,
      skip_screenshots: !update_screenshots,
      overwrite_screenshots: update_screenshots,
      app_version: version_number
    )

    commit_version_bump(
      xcodeproj: './ios/papertrailApp.xcodeproj',
      message: "build(ios): push #{version_number}: #{build_number} as release",
      force: true
    )

    add_git_tag

    push_to_git_remote
  end

  # You can define as many lanes as you want

  after_all do |lane|
    # This block is called, only if the executed lane was successful

    # clean_build_artifacts
    # slack(
    #   message: "Successfully deployed new App Update."
    # )
  end

  error do |lane, exception|
    # slack(
    #   message: exception.message,
    #   success: false
    # )
  end
end


# More information about multiple platforms in fastlane: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Platforms.md
# All available actions: https://docs.fastlane.tools/actions

# fastlane reports which actions are used. No personal data is recorded.
# Learn more at https://github.com/fastlane/fastlane#metrics
```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
app_identifier "com.papertrail.app" # The bundle identifier of your app
apple_id "ios@papertrail.io" # Your Apple email address

team_id "Z6Y4X6J5N8" # Developer Portal Team ID

# you can even provide different app identifiers, Apple IDs and team names per lane:
# More information: https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Appfile.md
```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.53.1  | ✅ Up-To-Date  |
| gym      | 2.0.0   | ✅ Up-To-Date  |
| pilot    | 2.0.0   | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.0.2        |
| executable-hooks          | 1.3.2        |
| bundler-unload            | 1.0.2        |
| rubygems-bundler          | 1.4.4        |
| bundler                   | 1.13.6       |
| io-console                | 0.4.6        |
| CFPropertyList            | 2.3.5        |
| i18n                      | 0.8.6        |
| minitest                  | 5.10.3       |
| thread_safe               | 0.3.6        |
| tzinfo                    | 1.2.3        |
| activesupport             | 4.2.9        |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.1        |
| babosa                    | 1.0.2        |
| claide                    | 1.0.2        |
| fuzzy_match               | 2.0.4        |
| nap                       | 1.1.0        |
| cocoapods-core            | 1.3.1        |
| cocoapods-deintegrate     | 1.0.1        |
| cocoapods-downloader      | 1.1.3        |
| cocoapods-plugins         | 1.0.0        |
| cocoapods-search          | 1.0.0        |
| cocoapods-stats           | 1.0.0        |
| netrc                     | 0.7.8        |
| cocoapods-trunk           | 1.2.0        |
| cocoapods-try             | 1.1.0        |
| colored2                  | 3.1.2        |
| escape                    | 0.0.4        |
| fourflusher               | 2.0.1        |
| gh_inspector              | 1.0.3        |
| molinillo                 | 0.5.7        |
| ruby-macho                | 1.1.0        |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.5.1        |
| cocoapods                 | 1.3.1        |
| colored                   | 1.2          |
| highline                  | 1.7.8        |
| commander-fastlane        | 4.4.5        |
| declarative               | 0.0.9        |
| declarative-option        | 0.1.0        |
| unf_ext                   | 0.0.7.4      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20170404 |
| dotenv                    | 2.2.1        |
| excon                     | 0.58.0       |
| multipart-post            | 2.0.0        |
| faraday                   | 0.13.1       |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| faraday_middleware        | 0.12.2       |
| fastimage                 | 2.1.0        |
| jwt                       | 1.5.6        |
| little-plugger            | 1.1.4        |
| multi_json                | 1.12.1       |
| logging                   | 2.2.2        |
| memoist                   | 0.16.0       |
| os                        | 0.9.6        |
| signet                    | 0.7.3        |
| googleauth                | 0.5.3        |
| httpclient                | 2.8.3        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| uber                      | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.1        |
| google-api-client         | 0.12.0       |
| json                      | 2.1.0        |
| mini_magick               | 4.5.1        |
| multi_xml                 | 0.6.0        |
| plist                     | 3.3.0        |
| rubyzip                   | 1.2.1        |
| security                  | 0.1.3        |
| slack-notifier            | 1.5.1        |
| terminal-notifier         | 1.8.0        |
| unicode-display_width     | 1.3.0        |
| terminal-table            | 1.8.0        |
| tty-screen                | 0.5.0        |
| word_wrap                 | 1.0.0        |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.2.8        |
| xcpretty-travis-formatter | 0.0.4        |
</details>

*generated on:* **2017-08-21**
</details>
```

**Comment by robwalkerco** [comment]
I dont seem to have any issues with the actual code signing itself. I can run `fastlane match appstore` successfully, and `match(type: "appstore")` runs fine in the lane. It seems to have issues during the `gym`phase.

**Comment by mpirri** [comment]
@robwalkerco I suspect the problem here is related to a older, standalone `gym` (and also `pilot`) gem you have installed. It is possible you have both `gym` and `pilot` in a Gemfile, or maybe you just need to `gem uninstall gem` and `gem uninstall pilot`.  Either way, you should not need to have either of these standalone gems installed, as the _fastlane_ gem has included those tools since version 2.0.0.

I hope this is helpful, please let us know if you have further questions!  🚀

**Comment by robwalkerco** [comment]
Thanks @mpirri. You hit the nail on the head 👏
